cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -124,6 +124,15 @@ pub enum FPUOp2 {
     Min64,
 }
 
+/// A floating-point unit (FPU) operation with two args, a register and an immediate.
+#[derive(Copy, Clone, Debug)]
+pub enum FPUOpRI {
+    UShr32(FPURightShiftImm),
+    UShr64(FPURightShiftImm),
+    Sli32(FPULeftShiftImm),
```

Comment:
Doc comment here (and I suppose on the other arms too) for those less familiar with AArch64: "shift left and insert", maybe briefly note the semantics (`Rd |= Rn << imm` I think?) too.